v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -72,13 +107,14 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
           "Properties string to use for the JDBC connection. Format of the string must be"
               + " [propertyName=property;]*.",
       example = "unicode=true;characterEncoding=UTF-8")
+  @Default.String("")
   String getSourceConnectionProperties();
 
   void setSourceConnectionProperties(String connectionProperties);
 
   @TemplateParameter.Text(
-      order = 5,
-      optional = true,
+      order = 8,
+      optional = false,
```

Comment:
For non use based auth, this will be optional.

Reply:
Fixed this in a previous push. Thanks.